test/tests/cat_tests.py

```diff
@@ -26,10 +26,22 @@ def test_cat(self):
         cat_cmd.errfile = io.StringIO()
 
         arg_parser = arguments.get_parser()
-        arg_sets = (['cat', str(test.id), 'build.sh'],)
+        arg_sets = (['cat', str(test.id), 'run.tmpl'],)
+        true_out="""
+#!/bin/bash
+
+# The first (and only) argument of the build script is the test id.
+export TEST_ID={}
+export PAV_CONFIG_FILE={}
+source {}
+
+# Perform the sequence of test commands.
+echo "Hello World."
+""".format(test.id, test.TODO, test.TODO)
```

Comment:
@Paul-Ferrell Where can I get the format field values from the quick test object?